server/routes/admin.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const express = require('express'),
+      router = express.Router(),
+      allEpisodeData = require('./all-episode-data');
+
+// return the state of enabled/disabled episodes
+// any episode not in this list is considered disabled by default
+router.get('/getEpisodes', function (req, res) {
+  return res.json(req.app.get('cache').getKey('episodes'));
```

Comment:
do we want to check the RSS feed for new episodes here?

Otherwise, I'm not totally clear on is when/how we check the RSS feed for new episodes—it seems this only happens when we boo the server or if we're in development mode and hit `/rss` on the Landing page from the frontend

Reply:
Hmmm, good point. As it stands the workflow would be: publish new episode, get your transcript in order, reboot the server, enable in admin panel. Probably shouldn't have to reboot the server every time. Maybe a button on the admin pane that says "refresh all episodes" which updates the episode cache from the live RSS.

Reply:
Actually, better idea, and closer to what you suggested: just update the cache itself here, since it's an admin pane. Just need to make sure it happens after authentication.